packages/x-charts/src/Gauge/GaugeContainer.tsx

```diff
@@ -77,6 +77,12 @@ const GaugeContainer = React.forwardRef(function GaugeContainer(props: GaugeCont
       aria-valuenow={value === null ? undefined : value}
       aria-valuemin={valueMin}
       aria-valuemax={valueMax}
+      style={
+        {
+          '--width': inWidth,
+          '--height': inHeight,
```

Comment:
```suggestion
          '--width': typeof inWidth === 'number' ? `${inWidth}px` : inWidth,
          '--height': typeof inHeight === 'number' ? `${inHeight}px` : inHeight,
```

as `width: 200` is invalid in CSS.